### ISSUE TITLE
Forbid to use alphabet as a string

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#895]
# Rule request

## Thesis

We should disallow strings with **full** alphabet inside it:

```python
GUESS_MY_NAME = "abcde...WXYZ"
UPPERCASE_ALPH = "ABCD...WXYZ"
LOWERCASE_ALPH = "abcd...wxyz"
```

Instead we should use:

```python
import string
UPPERCASE_ALPH = string.ascii_uppercase
LOWERCASE_ALPH = string.ascii_lowercase
```

## Reasoning

This is a best practice, similar to #787 
Original idea: https://github.com/thg-consulting/inspectortiger/blob/6b9736b22528493cd41d0bf92f9fdcdd6c7cd129/inspectortiger/plugins/upgradeable.py#L191

**Comment by PeterSherk** [comment]
Hello! Can I work on this?

**Comment by Amyth07** [comment]
Can you assign this issue to me?
